ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  SampleHandler.swift
+//  BroadcastExtension
+//
+//  Copyright © 2018 Twilio. All rights reserved.
+//
+
+import Accelerate
+import TwilioVideo
+import ReplayKit
+
+class SampleHandler: RPBroadcastSampleHandler, TVIRoomDelegate {
+
+    // Video SDK components
+    public var room: TVIRoom?
+    var videoSource: ReplayKitVideoSource?
+    var screenTrack: TVILocalVideoTrack?
+    let audioDevice = ExampleReplayKitAudioCapturer()
+
+    var accessToken: String = "TWILIO_ACCESS_TOKEN"
+    let accessTokenUrl = "http://127.0.0.1:5000/?identity=chris.ios&room=chris"
+
+    static let kBroadcastSetupInfoRoomNameKey = "roomName"
+
+    override func broadcastStarted(withSetupInfo setupInfo: [String : NSObject]?) {
+
+        TwilioVideo.audioDevice = ExampleReplayKitAudioCapturer(audioCapturer: self)
+
+        // User has requested to start the broadcast. Setup info from the UI extension can be supplied but is optional.
+        if (accessToken == "TWILIO_ACCESS_TOKEN" || accessToken.isEmpty) {
+            do {
+                accessToken = try TokenUtils.fetchToken(url: self.accessTokenUrl)
+            } catch {
+                let message = "Failed to fetch access token."
+                print(message)
+            }
+        }
+
+        // This source will attempt to produce smaller buffers with fluid motion.
+        videoSource = ReplayKitVideoSource(isScreencast: false)
+        let constraints = TVIVideoConstraints.init { (builder) in
+            builder.maxSize = CMVideoDimensions(width: Int32(ReplayKitVideoSource.kDownScaledMaxWidthOrHeight),
+                                                height: Int32(ReplayKitVideoSource.kDownScaledMaxWidthOrHeight))
+        }
+        screenTrack = TVILocalVideoTrack(capturer: videoSource!,
+                                         enabled: true,
+                                         constraints: constraints,
+                                         name: "Screen")
+
+        let localAudioTrack = TVILocalAudioTrack()
+        let connectOptions = TVIConnectOptions(token: accessToken) { (builder) in
```

Comment:
One issue here, we don't use the settings chosen in the main application. Should we be?